testsuite/features/core/srv_first_settings.feature

```diff
@@ -19,6 +19,7 @@ Feature: Very first settings
   As the admin user
   I want to create the organisation, the first users and set the HTTP proxy
 
+@skip_if_container_server
```

Comment:
Sadly this scenario seems to do more than just create a user.
What about:
`    And I run "rm -Rf /srv/salt/*" on "server"
`

Reply:
I wonder why we have that here! I will move it to a separate scenario to be sure

Reply:
Nice catch, Oscar. I would also prefer to have it in a different scenario.